DAO.sol

```diff
@@ -341,7 +362,7 @@ contract DAOInterface {
 
     /// @param _account The address of the account which is checked.
     /// @return Whether the account is blocked (not allowed to transfer tokens) or not.
-    function isBlocked(address _account) internal returns (bool);
+    function isBlocked(address _account) constant returns (bool);
```

Comment:
This is not a nice change, people should not be able to call isBlocked() as a constant, since it actually changes state.

We need to:
1. rename `isBlocked()` to `queryOrModifyBlocked()` since it can actually change the blocked state:
2. Create a constant `isBlocked()` function